camunda/client/external_task_client.py

```diff
@@ -137,12 +138,21 @@ def auth_basic(self) -> dict:
         token = AuthBasic(**self.config.get("auth_basic").copy()).token
         return {"Authorization": token}
 
+    @property
+    def auth_bearer(self) -> dict:
```

Comment:
pls add unit tests for this new function

Reply:
done @yogeshrnaik